src/Main.java

```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {

```

Comment:
String[] args parameter is not used.
method should return array of Strings, not void

Reply:
Done. Change `static void equationResult(String[] args)` to `static String[] equationResult(String[] args)`.